R/qsys_ssh.r

```diff
@@ -38,7 +38,7 @@ SSH = R6::R6Class("SSH",
             private$set_common_data(redirect=msg$proxy)
         },
 
-        submit_job = function(memory=NULL, log_worker=FALSE) {
+        submit_job = function(memory=NULL,walltime = NA,  log_worker=FALSE) {
```

Comment:
This also still misses the same modifications for `qsys_pbs` and `qsys_sge`, which will still fail with the same error.

Can you please also amend that?